loadtest/README.md

```diff
@@ -0,0 +1,111 @@
+ GET /api/users                                                   514     0(0.00%)  |     601     140    3289     420  |    0.61    0.00
+ GET /auth/profile                                                532     0(0.00%)  |     506     132    5066     330  |    0.64    0.00
+--------------------------------------------------------------------------------------------------------------------------------------------
+ Aggregated                                                      8109     0(0.00%)  |     630     132    5066     420  |    9.69    0.00
+
+Response time percentiles (approximated)
+ Type     Name                                                              50%    66%    75%    80%    90%    95%    98%    99%  99.9% 99.99%   100% # reqs
+--------|------------------------------------------------------------|---------|------|------|------|------|------|------|------|------|------|------|------|
+ GET      /api/cases                                                        450    660    830    960   1400   2000   3100   3500   3900   5000   5000   5033
+ GET      /api/sources                                                      400    570    740    830   1200   1700   2600   3000   3400   3500   3500   1032
+ GET      /api/sources/uploads                                              390    570    720    800   1100   1700   2500   2700   3200   3200   3200    998
+ GET      /api/users                                                        420    600    740    870   1300   1900   2500   2700   3300   3300   3300    514
+ GET      /auth/profile                                                     330    520    670    750   1100   1400   2100   2200   5100   5100   5100    532
+--------|------------------------------------------------------------|---------|------|------|------|------|------|------|------|------|------|------|------|
+ None     Aggregated                                                        420    630    780    900   1300   1900   2900   3300   3900   5100   5100   8109
+ ```
+
+![response time](screenshots/response_times_(ms)_1601386056.png)
+
+![total reqs](screenshots/total_requests_per_second_1601386056.png)
+
+![users](screenshots/number_of_users_1601386056.png)
+
+Data service resource usage at 10QPS did not increase significantly, CPU constraints seem to be on the curator service only.
+
+So basically: read-only traffic doesn't impact RAM usage much, curator service bottlenet is its CPU limits.
+
+## Initial setup
+
+Install Python 3.8, locust and the necessary dependencies with:
+
+```shell
+pip3 install -r requirements.txt
+```
+
+Get access to serialized credentials stored in S3 or generate your own and put them in an S3 bucket that you can access, then set the required environment variables when running locust:
+
+```shell
+S3_BUCKET='epid-ingestion' S3_OBJECT='covid-19-map-277002-0943eeb6776b.json'
+```
+
+You can check the [ingestion docs](/ingestion/functions/README.md) for how to generate/get those creds.
```

Comment:
Does it cover generating?

Reply:
it points to the aws docs, which covers generating yes.